db/migrate/20180210123431_create_topics.rb

```diff
@@ -0,0 +1,9 @@
+class CreateTopics < ActiveRecord::Migration[5.1]
+  def change
+    create_table :topics do |t|
+      t.string :body
```

Comment:
**Should a topic ever have an empty body?**

We don't have any model level validations yet (which is fine at the moment), but we should still ensure that dodgy data isn't getting into our database.  The technical term for ensuring this is "ensuring data integrity".

In addition to model validations we also have database level constraints. You can think of model validations as preventing _user_ rubbish getting into the database and database level constraints as preventing _developer_ rubbish. You can tell the database not to allow any null values (null == nil) in the migration as follows:

```
t.string :body, null: false
```

This will raise an error if you, the developer, ever cause this to happen.

Reply:
I don't like the name `body`. 

We should try to program using the language of the domain (i.e. the language of users and product owners), because then we don't need to remember the user translation that body means xxxxx: it's one less thing to remember.

Ask yourself would a user ever talk about a topic body? Wouldn't they be more likely to talk about a topic **description**? Happy to go with something else, but make it meaningful.